### DISCUSSION TITLE
Welcome to parler-py-api Discussions!

### DISCUSSION THREAD

**Discussion by KonradIT** [KonradIT/parler-py-api#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nvtuan96** [comment]
I want to intergate sharing feature with Parler.com to my website but they don't have any API. Could you help me?

**Comment by olli93jo** [comment]
Hey, I am looking for a tool/package for scraping some historical data for a specific hashtag from the Parler social network. Can I achieve this with this module?

**Reply by KonradIT** [reply]
Yes, in fact, look at experiment 2.

**Comment by MMMitchell14** [comment]
I am working on a research project, and we are trying to use Parler’s API. We have run into the issue that Parler is not interested in providing help or any type of API documentation. Do you know if Parler’s API has the ability to limit search based on geolocation. Within their private policy, they mention that they will track you if they need to, but I was wondering if outside coders could also access that information.

**Reply by KonradIT** [reply]
Not that I could find,

**Comment by KonradIT** [comment]
Well what a ride this was, hopefully they don't come back up.
